static/src/javascripts/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -31,7 +31,7 @@ const tests = [
     acquisitionsEpicLiveBlog,
     acquisitionsEpicAlwaysAskElection,
     acquisitionsEpicThankYou,
-].map(Test => new Test());
+];
```

Comment:
They're all still in `javascripts-legacy` but they all use the `contributions-utilities`, so don't need `new` anymore